jujugui/static/gui/src/app/app.js

```diff
@@ -929,6 +917,9 @@ YUI.add('juju-gui', function(Y) {
         return window.localStorage.getItem('discharge-token');
       };
       const charmstore = this.get('charmstore');
+      const setCreds = function(creds) {
```

Comment:
This works b/c the Login component no longer has the property at all. :smile:

I've deleted the whole bit.